### PR TITLE
Consistent usage of the CSS browser API

### PR DESCRIPTION
**Pull request by oliviertassinari**
Related to https://github.com/mui-org/material-ui/issues/16953. I'm not 100% convinced its the right solution, but it's worth trying.

*I hope you don't mind the created branch, I will delete it after.*

### CODE REVIEW

**Comment by kof** [comment]
I wonder what didn't work, in node this code shouldn't run?

**Comment by oliviertassinari** [comment]
I have tried to summarize the issue in this comment: https://github.com/mui-org/material-ui/issues/16953#issuecomment-520956695.

**Comment by kof** [comment]
yeah I mean how it comes `window.CSS === undefined`?

**Comment by oliviertassinari** [comment]
@kof It's curious. I don't know. At the same time, should we always use CSS or window.CSS. Is there a reason for mixing the two?

**Comment by kof** [comment]
`CSS` and `window.CSS` should be exactly the same, since window is the global. `window.CSS` is a more explicit way, I don't mind changing it the way you did it, but I don't understand what's going on, I fear there is some sort of confusion caused by something else, mb polyfills

**Comment by HenriBeck** [comment]
I think `CSS` should be safer because it tries to access `CSS` directly on the global object and not `window` on the global object which could be `undefined` and then it would crash.

**Comment by kof** [comment]
`window` IS the global object in a browser environment, it is safe, but I think something gets messed up on the way. Anyways I am merging it now.

**Comment by kof** [comment]
Forgot, @oliviertassinari ca you please add a changelog entry?

File: changelog.md [hunk 1]
```diff
@@ -13,6 +13,12 @@ Since you are interested in what happens next, in case, you work for a for-profi
 
 ---
 
+## 10.0.0-alpha.24 (2019-8-13)
```
Comment (kof) [inline changelog.md]: we have a Next headline which will be replaced with the release version at release time

Comment (kof) [reply changelog.md]: so below Next is the next release changes

File: packages/jss-plugin-default-unit/src/defaultUnits.js [hunk 2]
```diff
@@ -2,9 +2,9 @@
 
 import {hasCSSTOMSupport} from 'jss'
 
-const px = hasCSSTOMSupport ? window.CSS.px : 'px'
-const ms = hasCSSTOMSupport ? window.CSS.ms : 'ms'
-const percent = hasCSSTOMSupport ? window.CSS.percent : '%'
+const px = hasCSSTOMSupport && CSS ? CSS.px : 'px'
```
Comment (TrySound) [inline packages/jss-plugin-default-unit/src/defaultUnits.js]: You need to make sure CSS is defined or check with window.CSS
image

Comment (kof) [reply packages/jss-plugin-default-unit/src/defaultUnits.js]: hasCSSTOMSupport did that check already, I am just making flow happy

**Comment by kof** [comment]
released

**Comment by oliviertassinari** [comment]
Thanks for the release!

**Pull request merged**